jetbrains-core/tst/software/aws/toolkits/jetbrains/services/telemetry/TelemetryServiceTest.kt

```diff
@@ -190,10 +194,13 @@ class TelemetryServiceTest {
     }
 
     private fun assertMetricEventsContains(events: Collection<MetricEvent>, event: String, awsAccount: String, awsRegion: String) {
-        val metricEvent = events.find { e ->
+        events.find { e ->
```

Comment:
Doing 
```
    assertThat(events).anySatisfy {
            assertThat(it.data)...
        }
```

should print the whole list now

Reply:
```
        assertThat(events).anySatisfy {
            assertThat(it.data).extracting<String>(MetricEvent.Datum::name).contains(event)
            assertThat(it.awsAccount).describedAs("awsAccount").isEqualTo(awsAccount)
            assertThat(it.awsRegion).isEqualTo(awsRegion)
        }
```

yields:

```
java.lang.AssertionError: 
Expecting any element of:
  <[software.aws.toolkits.core.telemetry.DefaultMetricEvent@7fa760a1]>
to satisfy the given assertions requirements but none did:

  <software.aws.toolkits.core.telemetry.DefaultMetricEvent@7fa760a1> error: 
Expecting:
 <["Foo", "Faz", "Fuz", "Barr", "F"]>
to contain:
 <["Bar"]>
but could not find:
 <["Bar"]>
```

so we would need to implement the toStrings()s for Datum and MetricEvent, (which we prob should do...)

Reply:
That looks like a better way to do it, more clear what it does. Changed